egs2/interspeech2024_dsu_challenge/asr2/README.md

```diff
@@ -6,6 +6,47 @@ This can be referred to the ML-SUPERB [recipe](https://github.com/espnet/espnet/
 
 After download the dataset, please set the `MLSUPERB` to the data directory. The preparation will be automatically done in scripts for each tasks.
 
+# Bitrate calculation
```

Comment:
Please add the actual bitrate result to the baseline

Reply:
done.